src/index.js

```diff
@@ -1,11 +1,17 @@
 import Redis from 'redis-tag-cache';
+import crypto from 'crypto';
 
-export default function RedisCache (options) {
-  const client = new Redis(options);
+const defaultVersion = crypto.randomBytes(15).toString('hex')
 
+export default function RedisCache (options) {
+  const client = new Redis(options.redisConfig);
```

Comment:
```suggestion
  const { version, ...redisConfig } = options;
  const client = new Redis(redisConfig);
  const fallbackVersion = crypto.randomBytes(15).toString('hex');
  
  if (!version) {
    Logger.warn('The `version` property is missing in the `@vue-storefront/redis-cache` package configuration. In a multi-instance setup, this will result in a separate cache for every instance. Please refer to Redis driver documentation for more details.');
  }
```

Reply:
Done